src/train_model.py

```diff
@@ -1,17 +1,25 @@
-# This is for training a model
+import pickle
+import pandas as pd
 from sklearn.model_selection import train_test_split
-from sklearn.tree import DecisionTreeClassifier
+from sklearn.linear_model import LogisticRegression
 from sklearn.metrics import accuracy_score
+import config
 
-X, y = load_dataset()
+df = pd.read_csv(config.SOURCE)
 
-X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
+X = df.drop([config.TARGET], axis=1)
+y = df[config.TARGET]
 
-model = DecisionTreeClassifier()
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=0.3, random_state=42
+)
 
+model = LogisticRegression()
 model.fit(X_train, y_train)
 
 y_pred = model.predict(X_test)
 
-accuracy = accuracy_score(y_test, y_pred)
-print(f"Model Accuracy: {accuracy}")
+print("Logistic Regression \n")
+print("accuracy: ", accuracy_score(y_test, y_pred))
```

Comment:
this is only a suggestion, I know that I have been very nitpicky, but we should implement level logs. Even we could create a class for that. Here some examples of level logs:
DEBUG: used to debug
INFO: useful to provide information about an operation
WARNING: it's not an error, but it warns potential issues
ERROR: errors that needs attention
CRITICAL: critical error